src/swoop/api/workflows.py

```diff
@@ -1,7 +1,11 @@
-import yaml
 from fastapi import FastAPI
 
+from swoop.api.models.workflows import Workflows
+from swoop.api.routers.processes import create_workflows_dict
+
 
 def init_workflows_config(app: FastAPI) -> None:
     """Initialize Workflow Config."""
-    app.state.workflows = yaml.safe_load(open(app.state.settings.workflow_config_file))
+
+    wf = Workflows.from_yaml(app.state.settings.workflow_config_file)
+    app.state.workflows = create_workflows_dict(wf)
```

Comment:
Please see my PR into your branch for a better way to do this: https://github.com/Element84/swoop/pull/23.